aws-lc-rs/src/rsa/key.rs

```diff
@@ -456,6 +457,22 @@ where
     }
 }
 
+impl<B> TryInto<PublicEncryptingKey> for PublicKeyComponents<B>
```

Comment:
I'm not sure it's needed, but we could potentially have a non-consuming implementation:
```
impl<B> TryInto<PublicEncryptingKey> for &PublicKeyComponents<B>
...
``` 
Its use would be more cumbersome: `TryInto::try_into(&public_key)?` or `(&public_key).try_into()?`.